app/services/users/delete_comments.rb

```diff
@@ -2,7 +2,7 @@ module Users
   module DeleteComments
     module_function
 
-    def call(user, cache_buster = CacheBuster)
```

Comment:
This refactor is shaping up so nicely, it's great that we can get rid of this argument 🔥